apps/web/src/features/system-config/actions/getWinTags.ts

```
@@ -0,0 +1,350 @@
+            const formatDate = (dateStr: string) => {
+              // If the date string contains timezone info like "gmt+0200", remove it
+              // and ensure we have a proper ISO date string
+              try {
+                // Parse the date and convert to ISO string (UTC)
+                const date = new Date(dateStr);
+                return date.toISOString();
+              } catch {
+                // If parsing fails, return the original value
+                return dateStr;
+              }
+            };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)